crates/bevy_asset/src/asset_server.rs

```diff
@@ -513,6 +513,14 @@ impl AssetServer {
             }
         }
     }
+
+    pub fn is_directory<P: AsRef<Path>>(&self, path: P) -> bool {
+        self.server.asset_io.is_directory(path.as_ref())
+    }
+
+    pub fn is_file<P: AsRef<Path>>(&self, path: P) -> bool {
+        self.server.asset_io.is_file(path.as_ref())
+    }
```

Comment:
I'm not a fan of this on the `AssetServer` class, as it seems to break the _single_ responsibility of the struct.

Reply:
Yeah rather than mirror AssetIo methods here, we should probably just add an `asset_io` accessor method to AssetServer (no need for `asset_io_mut` because it currently has no mut methods).